From fce671c791da12bbd64666d054e81948e05dfb0b Mon Sep 17 00:00:00 2001
From: Stefan Lippers-Hollmann <s.l-h@gmx.de>
Date: Mon, 20 Jan 2014 18:18:14 +0100
Subject: [PATCH] Revert "Bluetooth: Use devname:vhci module alias for virtual
 HCI driver"

This reverts commit bfacbb9aec029b3200053d84c8cd5d7575f2d4a5, thereby
avoiding:

depmod: ERROR: Module 'hci_vhci' has devname (vhci) but lacks major and
minor information. Ignoring.

Signed-off-by: Stefan Lippers-Hollmann <s.l-h@gmx.de>
---
 drivers/bluetooth/hci_vhci.c | 1 -
 1 file changed, 1 deletion(-)

--- a/drivers/bluetooth/hci_vhci.c
+++ b/drivers/bluetooth/hci_vhci.c
@@ -377,4 +377,3 @@ MODULE_AUTHOR("Marcel Holtmann <marcel@h
 MODULE_DESCRIPTION("Bluetooth virtual HCI driver ver " VERSION);
 MODULE_VERSION(VERSION);
 MODULE_LICENSE("GPL");
-MODULE_ALIAS("devname:vhci");
